EIPS/eip-TBD.md

```diff
@@ -1,85 +1,57 @@
 ---
-title: NFT royalties with on-chain, decentralized enforcement
-description: TBD
-author: David Huber (@cxkoda) and Arran Schlosberg (@aschlosberg)
+title: On-chain NFT royalty enforcement
+description: Hello world
+author: David Huber (@cxkoda), Arran Schlosberg (@aschlosberg)
 discussions-to: <URL>
 status: Draft
 type: Standards Track
 category: ERC
 created: 2023-09-25
-requires: 165, 721, 2981
+requires: 165, 721
 ---
 
-<!--
-  READ EIP-1 (https://eips.ethereum.org/EIPS/eip-1) BEFORE USING THIS TEMPLATE!
-
-  This is the suggested template for new EIPs. After you have filled in the requisite fields, please delete these comments.
-
-  Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`.
-
-  The title should be 44 characters or less. It should not repeat the EIP number in title, irrespective of the category.
-
-  TODO: Remove this comment before submitting
--->
-
 ## Abstract
 
-We describe a mechanism for which, upon transfer of a Non-Fungible Token (NFT), the dominant strategy of the new owner is to truthfully reveal if the owning entity has changed and, if so, report their valuation of the token and pay a creator-specified royalty based on said valuation.
-
-<!--
-  The Abstract is a multi-sentence (short paragraph) technical summary. This should be a very terse and human-readable version of the specification section. Someone should be able to read only the abstract to get the gist of what this specification does.
-
-  TODO: Remove this comment before submitting
--->
+We describe a mechanism for which, upon transfer of an [ERC-721](./eip-721.md) Non-Fungible Token (NFT), the dominant strategy of the new owner is to truthfully reveal if the owning entity has changed and, if so, (a) report their valuation of the token and (b) pay a creator-specified royalty based on said valuation.
```

Comment:
Why the a) b)? I'd remove since we never need to reference.

Reply:
Done.

I tend to put them in in case others want to reference them, but I agree that it's unnecessary here.